ERCS/erc-7562.md

```diff
@@ -0,0 +1,393 @@
+---
+eip: 7562
+title: Account Abstraction Validation Scope Rules
+description: A set of limitations on validation EVM code to protect Account Abstraction nodes from denial-of-service attacks through unpaid computation.
+author: Yoav Weiss (@yoavw), Dror Tirosh (@drortirosh), Alex Forshtat (@forshtat), Shahaf Nacson (@shahafn)
+discussions-to: https://ethereum-magicians.org/t/erc-7562-account-abstraction-validation-scope-rules/16683
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-09-01
+---
```

Comment:
These proposals seem like they'd be important to read before implementing the rules referencing them (OP-70 / OP-62).

```suggestion
requires: 1153, 7212
---
```

Reply:
both 1153 and 7212 are optional. 
We do declare the validation rules in their presence, but the rules are valid even on networks they are missing, so "requires" seems like a strong word. Is there an equivalent "optional" ?

Reply:
Dror, I know it's a bit confusing but `requires` here means "to understand this EIP/ERC, it requires the other EIP/ERC listed here". It doesn't mean the other EIP/ERCs listed in `requires` is a Specification-wise dependency.

It seems in the context 1153 and 7212 are a fit for the `requires`. I'd also include 4337. 

That said, I won't consider having or not having these `requires` a blockage for merge as `Status: Draft`

Reply:
I can't include erc-4337, since 4337 actually requires this one, and it would cause a circular dependency.
I strongly suggest that if "requires" means "it is a mere reference for understanding this erc", then it should be renamed accordingly.
(btw: the HTML processor could generate these "references" automatically. an editor should explicitly specify what is really "required".